.golangci.yml

```diff
@@ -4,17 +4,23 @@
 # options for analysis running
 run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 1m
+  timeout: 15m
 
 issues:
   # Maximum count of issues with the same text.
   # Set to 0 to disable.
   # Default: 3
-  max-same-issues: 0
+  max-same-issues: 3
   # Maximum issues count per one linter.
   # Set to 0 to disable.
   # Default: 50
-  max-issues-per-linter: 0
+  max-issues-per-linter: 50
+  exclude-rules:
+    # Exclude package name contains '-' issue because we have at least one package with
+    # it on its name.
+    - text: "ST1003:"
```

Comment:
You could just fix this one in this repository, unless you specifically want to keep allowing it

Reply:
I'll fix it ;)

As I mentioned on the other comment, I didn't pay much attention to the contents, just made sure the linter was working.